### PR TITLE
Show link to wiki when AzCopy fails (with no explanation) on Linux

### PR DESCRIPTION
**Pull request by wwlorey**
Related: https://github.com/microsoft/vscode-azurestorage/issues/917

After looking into it more, I was hesitant to auto-detect this package because you have to use that distro's package manager to query the installed packages. This varies based on the distro and keeping all of those commands up to date sounds like a can 'o worms 🪱

### CODE REVIEW

**Comment by nturinski** [comment]
Were you passing which into the shell?  Have you looked into using this package? https://www.npmjs.com/package/which

**Comment by wwlorey** [comment]
> Were you passing which into the shell? Have you looked into using this package? https://www.npmjs.com/package/which

Unfortunately `which` doesn't detect if this package is installed (I guess it's not in the path.) Here's an example of what querying the package database looks like on Kali with `libsecret-1-dev` installed:

Screen Shot 2021-01-28 at 1 27 32 PM

File: README.md [hunk 1]
```diff
@@ -62,6 +62,16 @@ Use these configuration settings to customize the Azure Storage extension.
 * `azureStorage.file.showSavePrompt`: Set to `false` to prevent showing a warning dialog on File file save.
 * `azureStorage.blob.showSavePrompt`: Set to `false` to prevent showing a warning dialog on Blob file save.
 
+## Known Issues
+
+* `libsecret-1-dev` is required for AzCopy transfers on Linux and is not installed by default in many distros running in WSL.
```
Comment (ejizba) [inline README.md:67]: Is this how Storage Explorer handles this as well? Just by documenting it?

Comment (ejizba) [reply README.md:67]: > you have to use that distro's package manager to query the installed packages

Fwiw, I agree this sounds like a can of worms not worth opening. Just curious how SE handles it

Comment (wwlorey) [reply README.md:67]: They include libsecret as a dependency when creating their snap package. I also looked at the AzCopy repo and didn't find any guidance on downloading required dependencies on WSL

Comment (ejizba) [reply README.md:67]: And I assume that doesn't help us at all? Can we check telemetry and see how frequent this problem is? It seems like a big deal considering how much we rely on AzCopy now.

In terms of "ideas that don't actually fix it", what if we catch the error on Linux and point them to a "Troubleshooting" page or something (could just be in our GitHub Wiki).

Comment (nturinski) [reply README.md:67]: Yeah, I prefer to have at least something like that.  I feel like the visibility on the README is pretty low as in users aren't likely to look at it.

Comment (wwlorey) [reply README.md:67]: I agree that the README visibility isn't great. I'll add a add a troubleshooting notification on linux if we see the AzCopy failed error

**Comment by wwlorey** [comment]
Here's what the AzCopy fail error looks like on linux:

Screen Shot 2021-01-29 at 3 30 17 PM

File: src/commands/azCopy/azCopyTransfer.ts [hunk 2]
```diff
@@ -75,6 +75,10 @@ async function handleJob(context: IActionContext, jobInfo: IJobInfo, transferLab
             // tslint:disable-next-line: no-floating-promises
             ext.ui.showWarningMessage(message);
         } else {
+            if (/^AzCopy Transfer: "Failed"\.\s$/i.test(message) && process.platform === 'linux') {
+                message += localize('viewTheWiki', 'View [the wiki](https://github.com/microsoft/vscode-azurestorage/wiki/Known-Issues) for help with known issues.');
```
Comment (ejizba) [inline src/commands/azCopy/azCopyTransfer.ts:79]: The message here should be more about _what_ they should should view rather then _where_ they should view it. Who knows, maybe we move the content out of the wiki!

I'd suggest
```
View help with [known issues]().
```

Or
```
View [troubleshooting tips]().
```

Comment (ejizba) [inline src/commands/azCopy/azCopyTransfer.ts:79]: Use an aka.ms link

**Review by ejizba — APPROVED** [review]

**Pull request merged**